roles/lib_openshift/library/oc_secret.py

```diff
@@ -1609,6 +1620,8 @@ def create(self, files=None, contents=None):
 
         secrets = ["%s=%s" % (sfile['name'], sfile['path']) for sfile in files]
         cmd = ['secrets', 'new', self.name]
+        if self.type is not None:
+            cmd.append("--type=%s" % (self.type))
```

Comment:
I recommend using ``"".format(..)`` instead of ``%s`` but it's not worth changing right now 😄